config/components.yml

```diff
@@ -26,9 +26,14 @@ containers:
   - "JavaBuildpack::Container::Tomcat"
 
 # In order to use Oracle JREs instead of OpenJDK, you must comment out the OpenJDK line and uncomment the Oracle line.
+# In order to use Zulu JREs instead of OpenJDK, you must comment out the OpenJDK line and uncomment the Zulu line.
+# To use Zulu JRE instead of OpenJDK without forking java-buildpack, set environment variable:
```

Comment:
@Ekaterina001 This isn't the proper location for documentation of this level of detail.  It's generally covered [in the README](https://github.com/cloudfoundry/java-buildpack#configuration-and-extension), but if you'd like Zulu-specific documentation, please add it to the main Zulu documentation page.